docker/Dockerfile

```
@@ -169,20 +174,22 @@ RUN set -ex; \
   { \
     echo "[network]"; \
     echo "network = '${NETWORK}'"; \
+    [ $NETWORK = "Testnet" ] && echo "listen_addr = '127.0.0.1:18233'"; \
```

This is not required, the Rust deserialization code already uses the correct port for the network:
https://github.com/ZcashFoundation/zebra/blob/f1bedafbbb1bab97b2536fee50062db47bc4bbea/zebra-network/src/config.rs#L332

So either of these should work:

```suggestion
```

```suggestion
    echo "listen_addr = '127.0.0.1'"; \
```

---

Actually, I think there might be a bug in the deserializer code:
https://github.com/ZcashFoundation/zebra/blob/f1bedafbbb1bab97b2536fee50062db47bc4bbea/zebra-network/src/config.rs#L312-L324

I think the default deserialized listen address needs to be "0.0.0.0", so that we get the correct default port when the address is missing. This should probably be a separate PR.

---

Using an address of "127.0.0.1" in the config should still work though!

---

Let's open another issue for the config fix, and just use an address in this one?

That way we can schedule the config fix based on our priorities, and get this PR merged without needing any Zebra fixes.

---

I opened this ticket for the Rust default testnet port config fix:
- #6386